.github/workflows/go.yml

```diff
@@ -49,6 +49,9 @@ jobs:
     - name: Vet
       if: matrix.os == 'ubuntu-latest'
       run: make vet
+    - name: Coverage
+      if: matrix.os == 'ubuntu-latest'
+      run: go test -coverprofile=cover.out ./...
```

Comment:
i think this step is not needed. you can just add `-coverprofile=cover.out` to the test step below (Test with Race detector). the coveralls only run on the ubuntu build (the one that uses the test with race detector). not sure if `make` will like the extra params though, we might need to change the makefile

Reply:
@acud I thought that but `make` doesn't accept additional argument (unless I do some changes) and we will create `cover.out` on mac as well (which is not a real issue), if we do it like this tests will be cached and we will not slow down whole pipeline

Reply:
cool :+1: